ansible_ai_connect/ai/api/model_client/tests/test_wca_client.py

```diff
@@ -1183,43 +1187,56 @@ def test_get_model_id_without_setting(self):
 @override_settings(ANSIBLE_AI_MODEL_MESH_MODEL_ID="model-name")
 @override_settings(ANSIBLE_AI_MODEL_MESH_API_TIMEOUT=None)
 class TestWCAOnPremCodegen(WisdomServiceLogAwareTestCase):
-    def test_headers(self):
-        suggestion_id = "suggestion_id"
-        prompt = "- name: install ffmpeg on Red Hat Enterprise Linux"
-        token = base64.b64encode(bytes("username:12345", "ascii")).decode("ascii")
-
-        model_input = {
-            "instances": [
-                {
-                    "context": "",
-                    "prompt": prompt,
-                }
-            ]
-        }
-        codegen_data = {
-            "model_id": "model-name",
-            "prompt": f"{prompt}\n",
-        }
-
-        request_headers = {
-            "Authorization": f"ZenApiKey {token}",
-            WCA_REQUEST_ID_HEADER: suggestion_id,
-        }
-
-        model_client = WCAOnPremClient(inference_url="https://example.com")
-        model_client.session.post = Mock(return_value=MockResponse(json={}, status_code=200))
+    prompt = "- name: install ffmpeg on Red Hat Enterprise Linux"
+    suggestion_id = "suggestion_id"
+    token = base64.b64encode(bytes("username:12345", "ascii")).decode("ascii")
+    model_input = {
+        "instances": [
+            {
+                "context": "",
+                "prompt": prompt,
+            }
+        ]
+    }
+    codegen_data = {
+        "model_id": "model-name",
+        "prompt": f"{prompt}\n",
+    }
+    request_headers = {
+        "Authorization": f"ZenApiKey {token}",
+        WCA_REQUEST_ID_HEADER: suggestion_id,
+    }
+    model_client = WCAOnPremClient(inference_url="https://example.com")
 
-        model_client.infer(
+    def test_headers(self):
```

Comment:
Should you not add `@override_settings(ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL=True)`?

If somebody runs this test locally with the setting `False` the test will fail.